connectors/aws/aws-sqs/element-templates/aws-sqs-boundary-connector.json

```diff
@@ -155,7 +155,8 @@
     "constraints" : {
       "notEmpty" : true,
       "pattern" : {
-        "value" : "^([0-9]?|1[0-9]|20|secrets\\..+)$"
+        "value" : "^(secrets|([0-9]?|1[0-9]|20)).*$",
+        "message" : ""
```

Comment:
Here is the pattern for the modeler. FeelMode is disabled here, no variables can be passed and 'null' as a user value could not be inserted as it does not match the pattern

Reply:
I was more wondering about the error message. I knew there was a default message when there is nothing custom provided in the template (`<propertyId> has to match the expression <provided expression>)`. So I was curious if the same message would be displayed if we provide an empty string, or if it would just display the empty string as is. I checked it in the modeler and it looks like the behavior is identical and the default message is shown in both cases, so I think we are good 👍